Source/JavaScriptCore/b3/air/AirOpcode.opcodes

```diff
@@ -147,6 +147,9 @@ x86: Add16 U:G:16, UD:G:16
     Imm, Tmp, Tmp
     Tmp, Tmp, Tmp
 
+32: Add64 U:G:32, U:G:32, U:G:32, U:G:32, D:G:32, D:G:32
+    Tmp, Tmp, Tmp, Tmp, Tmp, Tmp
+
```

Comment:
This is weird to me. I feel like whoever is emitting Air code should be emitting the instructions they need. Instead of us pretending there is an Add64 on 32-bit CPUs

Reply:
Yeah, agreed it's a little awkward. Hopefully we can talk more about what we think a good alternative would look like; if we choose to lift the add-with-carry instruction up to air, we'll just need to teach air about the carry flag, which I think at present, flags are never live between instructions, right?